src/ServicePulse.Host/app/modules/configuration/js/license/license.module.js

```diff
@@ -3,11 +3,8 @@
 
     angular.module('configuration.license', []);
 
-    require('./license.controller');
     require('./license.route');
```

Comment:
Swapped around because reasons. Not really sure why this needs to be in this order, but...